src/utils/convert.js

```diff
@@ -83,20 +83,32 @@ ${props.join(',\n')}
 };
 
 const renderSampleField = (def) => {
-  const type = typesMap[def.type];
+  const type = typesMap[def.type] || 'string';
 
-  return `      ${def.key}: {
+  if (def.label) {
+    return `      {
+        key: ${quote(def.key)},
         type: ${quote(type)},
         label: ${quote(def.label)}
       }`;
+  }
+
+  return `      {
+        key: ${quote(def.key)},
+        type: ${quote(type)}
+      }`;
 };
 
 const renderSample = (definition) => {
   const fields = _.map(definition.sample_result_fields, renderSampleField);
 
-  return `    sample: {
+  if (!fields || fields.length === 0) {
```

Comment:
Looks like lodash will always return an array. 

```js
if (!fields.length) {
  return '';
}
```

Any particular reason we're not using `JSON.dump`?